maintainers/maintainer-list.nix

```diff
@@ -19159,6 +19159,12 @@
     githubId = 18403034;
     name = "Shaddy";
   };
+  shadowapex = {
```

Comment:
Should ideally be in an isolated commit.

Reply:
Separated in commit: https://github.com/NixOS/nixpkgs/pull/344346/commits/f3ff82fc64ca115215e1c618ae993f94f6b6fd16